src/components/BioButton/styles.scss

```diff
@@ -1,4 +1,5 @@
 @import "node_modules/@biotope/element/lib/host.mixin";
+@import '../../resources/scss/defaultSassImports.scss';
 @import "../../resources/scss/settings/settings";
```

Comment:
if you include defaultSassImports.scss please remove the deprecated settings/settings, itis included in the sassImports